ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/DropSourceCommandTest.java

```diff
@@ -71,13 +57,12 @@
   public void setUp() {
     when(metaStore.getSource(STREAM_NAME)).thenReturn((StructuredDataSource)dataSource);
     when(dataSource.getDataSourceType()).thenReturn(DataSourceType.KSTREAM);
-    when(dataSource.getKafkaTopicName()).thenReturn(TOPIC_NAME);
   }
 
   @Test
   public void shouldSucceedOnMissingSourceWithIfExists() {
     // Given:
-    givenDropSourceCommand(IF_EXISTS, WITH_DELETE_TOPIC);
+    givenDropSourceCommand(IF_EXISTS, WITHOUT_DELETE_TOPIC);
```

Comment:
In your current code `DropSourceCommand` shouldn't even take a boolean indicating if the topic should be deleted.  Hence drop this second param.

Caveat: see concern about backwards compatibility in main review text.